content/guides/create-a-pull-request/index.md

```diff
@@ -0,0 +1,134 @@
+---
+id: creating-a-pull-request
+title: Creating a Pull Request
+emoji: 📋
+---
+
+# 🔃 Comparing changes
```

Comment:
```suggestion
## 🔃 Comparing changes
```

So it actually shows up in the table of contents - right now there's an empty TOC entry for bullet 1 (I'm not sure why)